deployment/pom.xml

```diff
@@ -9,16 +9,137 @@
   </parent>
   <artifactId>quarkus-openapi-generator-deployment</artifactId>
   <name>Quarkus - Openapi Generator - Deployment</name>
+
+  <properties>
+    <version.org.openapitools>5.4.0</version.org.openapitools>
+    <version.org.slf4j>1.7.29</version.org.slf4j>
+    <version.com.github.jknack>4.2.0</version.com.github.jknack>
+    <version.io.swagger>1.6.2</version.io.swagger>
+    <version.io.swagger.core.v3>2.1.5</version.io.swagger.core.v3>
+  </properties>
+
   <dependencies>
+    <!-- Quarkus -->
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-arc-deployment</artifactId>
+      <artifactId>quarkus-core-deployment</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-rest-client-jackson-deployment</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-devtools-utilities</artifactId>
+      <!-- Maybe we need to add it to the quarkus bom -->
+      <version>${quarkus.version}</version>
```

Comment:
https://github.com/quarkusio/quarkus/commit/7526f31d3386517f351fff7616e841f406acec5e introduced it in Quarkus 2.7.1.Final